File: app/src/main/java/com/asfoundation/wallet/analytics/AnalyticsBody.java
package com.asfoundation.wallet.analytics;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.Map;

public class AnalyticsBody {

  @JsonProperty("aptoide_vercode") private int appcoinsVercode;
  @JsonProperty("aptoide_package") private String
      appcoinsPackage;
  private Map data;

  public AnalyticsBody(int appcoinsVercode, String appcoinsPackage, Map data) {
    this.appcoinsVercode = appcoinsVercode;
    this.appcoinsPackage = appcoinsPackage;
    this.data = data;
  }

  public int getAppcoinsVercode() {
    return appcoinsVercode;
  }

  public String getAppcoinsPackage() {
    return appcoinsPackage;
  }

  public Map getData() {
    return data;
  }
}

File: core/network/analytics/src/main/java/com/appcoins/wallet/core/network/analytics/AnalyticsBody.kt
package com.appcoins.wallet.core.network.analytics

import com.fasterxml.jackson.annotation.JsonProperty

class AnalyticsBody(
  @field:JsonProperty("aptoide_vercode")
  val appcoinsVercode: Int, @field:JsonProperty("aptoide_package")
  val appcoinsPackage: String, val data: Map<*, *>
)